topics/galaxy-interface/tutorials/workflow-fairification/tutorial.md

```diff
@@ -0,0 +1,668 @@
+
+Applying these principles to workflows is equally important for good data management:
+
+- **Enhanced Discoverability**: Well-annotated and documented workflows are easier to find, making them more likely to be used and cited.
+- **Improved Reproducibility**: Standardized and tested workflows ensure that analyses can be reproduced, validating research findings.
+- **Community Collaboration**: Sharing workflows through centralized registries fosters collaboration and innovation within the bioinformatics community.
+- **Sustainability**: Regular updates and versioning ensure that workflows remain compatible with the latest tools and Galaxy releases, extending their lifespan and utility.
+
+While making a workflow FAIR might seem complicated ({% cite wilkinson2025applying %}, {% cite goble2020fair %}), publications like "Ten quick tips for building FAIR workflows" ({% cite de2023ten %}) provide practical guidelines to simplify the process:
+
+![This image provides ten quick tips for building FAIR workflows, focusing on findability (registering the workflow and describing it with rich metadata), accessibility (making source code available in a public repository and providing example input data and results), interoperability (adhering to file format standards and making the workflow portable), and reusability (providing a reproducible computational environment, adding a configuration file with defaults, modularizing the workflow, and offering clear documentation).](./images/journal.pcbi.1011369.g001.PNG "Ten quick tips for building FAIR workflows. Source: {% cite de2023ten %}")
+
+Using Galaxy as a workflow management system, many of these tips (all related to **Interoperability** and **Reusability**) are already fulfilled:
+- Tip 5 (**Interoperability**): *"The tools integrated in a workflow should adhere to file format standards."*
+- Tip 6 (**Interoperability**): *"Make the workflow portable."*
+- Tip 7 (**Reusability**): *"Provide a reproducible computational environment to run the workflow."*
+- Tip 8 (**Reusability**): *"Add a configuration file with defaults."*
+- Tip 9 (**Reusability**): *"Modularize the workflow."*
+
+In this tutorial, we will demonstrate how to fulfill the remaining tips:
+- Tip 1 (**Findability**): *"Register the workflow."*
+- Tip 2 (**Findability**): *"Describe the workflow with rich metadata."*
+- Tip 3 (**Accessibility**): *"Make source code available in a public code repository."*
+- Tip 4 (**Accessibility**): *"Provide example input data and results along with the workflow."*
+- Tip 10 (**Reusability**): *"Provide clear and concise workflow documentation."*
+
+To illustrate the process, we will use a simple workflow with 2 steps as an example.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Prepare the workflow
+
+Here, we will use a workflow running FastQC and MultiQC but you can use your own workflow.
```

Comment:
Can we use Falco instead of FastQC? Or Fastp?

Reply:
Can Falco output be used in MultiQC? I can not find Falco in the list of tools in MultiQC

Reply:
Will try then